File: Kattis/TakeTwoStones.java
package taketwostones;

import java.util.Scanner;

public class TakeTwoStones {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int input = sc.nextInt();
        
        if (input % 2 == 0) {
            System.out.println("Bob");
        } else {
            System.out.println("Alice");
        }
    }
    
}

File: Kattis/CPlusPlus/TakeTwoStones.cpp
#include <iostream>

int main() {
    int input;

    std::cin >> input;

    if (input % 2 == 0) {
        std::cout << "Bob" << std::endl;
    } else {
        std::cout << "Alice" << std::endl;
    }

    system("PAUSE");
    return 0;
}